synapse/cortex.py

```diff
@@ -538,15 +537,14 @@ class Cortex(s_cell.Cell):
 
         await self.addCoreMods(s_modules.coremods)
 
+        self.triggers = s_trigger.Triggers(self)
```

Comment:
(Preface: all this startup ordering stuff is getting dicey).
New problematic order: mod1 causes trigger to execute that has a form from mod2, that isn't loaded yet. will blow up.

Reply:
i split mod loading a bit so all models from all mods are loaded before here and they are all initCoreModule()'d after this.  no more issue :)

Reply:
This is still an issue since a third party module could load a storm command that a trigger depends on.  Since the loaded commands are part of the storm syntax, we could fail to parse a stored query on restart.